marimo/_plugins/stateless/json_output.py

```diff
@@ -7,25 +7,38 @@
 from marimo._plugins.core.web_component import JSONType, build_stateless_plugin
 
 
-def json_output(json_data: JSONType, name: Optional[str] = None) -> Html:
+def json_output(
+    json_data: JSONType,
+    name: Optional[str] = None,
+    value_types: Optional[str] = None,
+) -> Html:
     """Build a json output element.
 
     Args:
     -----
     json_data: JSON-serializable data to display
     name: optional text label
+    value_types: optional value types to display,
+        e.g. "python" (default) or "json"
 
     Returns:
     --------
     A string of HTML for a JSON output element.
     """
+    element_args = {
+        "json-data": json_data,
+    }
```

Comment:
doing the above would simplify:
    element_args = {
        "json-data": json_data,
        "value-types": value_types,
    }